pkg/kernel/kernel-config/kernel_config-5.10.x-preempt-rt-x86_64.patch

```diff
@@ -0,0 +1,195 @@
+--- kernel_config-5.10.x-x86_64	2023-05-24 14:58:44.753098744 +0200
++++ kernel_config-5.10.x-x86_64	2023-05-24 13:55:04.964388018 +0200
+@@ -98,9 +98,14 @@
+ CONFIG_HIGH_RES_TIMERS=y
+ # end of Timers subsystem
+ 
++CONFIG_HAVE_PREEMPT_LAZY=y
++CONFIG_PREEMPT_LAZY=y
+ # CONFIG_PREEMPT_NONE is not set
+-CONFIG_PREEMPT_VOLUNTARY=y
++# CONFIG_PREEMPT_VOLUNTARY is not set
+ # CONFIG_PREEMPT is not set
++CONFIG_PREEMPT_RT=y
++CONFIG_PREEMPT_COUNT=y
++CONFIG_PREEMPTION=y
+ 
+ #
+ # CPU/Task time and stats accounting
+@@ -123,14 +128,18 @@
+ # RCU Subsystem
+ #
+ CONFIG_TREE_RCU=y
++CONFIG_PREEMPT_RCU=y
+ # CONFIG_RCU_EXPERT is not set
+ CONFIG_SRCU=y
+ CONFIG_TREE_SRCU=y
+ CONFIG_TASKS_RCU_GENERIC=y
++CONFIG_TASKS_RCU=y
+ CONFIG_TASKS_RUDE_RCU=y
+ CONFIG_TASKS_TRACE_RCU=y
+ CONFIG_RCU_STALL_COMMON=y
+ CONFIG_RCU_NEED_SEGCBLIST=y
++CONFIG_RCU_BOOST=y
++CONFIG_RCU_BOOST_DELAY=500
+ # end of RCU Subsystem
+ 
+ CONFIG_BUILD_BIN2C=y
+@@ -161,7 +170,6 @@
+ CONFIG_CGROUP_SCHED=y
+ CONFIG_FAIR_GROUP_SCHED=y
+ CONFIG_CFS_BANDWIDTH=y
+-CONFIG_RT_GROUP_SCHED=y
+ CONFIG_CGROUP_PIDS=y
+ # CONFIG_CGROUP_RDMA is not set
+ CONFIG_CGROUP_FREEZER=y
+@@ -255,10 +263,10 @@
+ # end of Kernel Performance Events And Counters
+ 
+ CONFIG_VM_EVENT_COUNTERS=y
++# CONFIG_SLUB_DEBUG is not set
++# CONFIG_SLUB_MEMCG_SYSFS_ON is not set
+ # CONFIG_COMPAT_BRK is not set
+-CONFIG_SLAB=y
```

Comment:
I didn't notice we were using SLAB before.... but I guess you are changing to SLUB only for improvements, right?

Reply:
This is the default, comes with the preemt_rt. I left untouched.